tests/network/localnet/migration_stuntime/test_migration_stuntime.py

```
@@ -19,49 +19,46 @@
+            pytest.param(6, marks=[pytest.mark.ipv6, pytest.mark.polarion("CNV-15272")], id="ipv6"),
         ],
         indirect=True,
+        scope="class",
     )
-    def test_client_migrates_off_server_node(
-        self,
-        admin_client,
-        localnet_stuntime_client_vm,
-        active_ping,
-    ):
+    def test_client_migrates_off_server_node(self, admin_client, ip_family, localnet_stuntime_client_vm, active_ping):
```

Ruff (0.15.13)

[warning] Unused method argument: `ip_family`

(ARG002)